src/web/recorder/recorder.tsx

```diff
@@ -42,6 +44,13 @@ export const Recorder: React.FC<RecorderProps> = ({
   log,
   mode
 }) => {
+  const [selector, setSelector] = React.useState('');
```

Comment:
Recorder can't have state, it won't be storybookable then, you should move state into main.tsx.